setup.cfg

```diff
@@ -45,12 +45,3 @@ exclude =
 
 [bdist_wheel]
 universal = 1
-
-[flake8]
-max-complexity = 10
-max-line-length = 119
-exclude = .venv,djstripe/migrations/,.tox
-
-[pycodestyle]
-max-line-length = 119
-exclude = .venv,djstripe/migrations/,.tox
```

Comment:
> Also, if I were to run tests without tox to quickly check if I've broken anything, flake8 won't look at tox.ini (I assume)

It will; it looks in `.flake8`, `tox.ini` and `setup.cfg` in that order.

What does linty use exactly? The pycodestyle section?